src/routes/AccountDetailsContainer.js

```diff
@@ -87,13 +94,49 @@ class AccountDetailsContainer extends React.Component {
     return groups.filter(g => g.id === user.patronGroup)[0] || {};
   }
 
+  getItemDetails = () => {
+    const account = this.getAccount();
+    const { resources } = this.props;
+    const loanRecord = _.get(resources, ['loans', 'records'], []);
```

Comment:
Could you please clarify if the syntax from the line 103 'const contributorsRecord = item[0]?.item?.contributors ?? [];' can be applied here for the consistency?

And could you please check changes in other files? For example, AccountDetails.js.